app/src/main/java/com/github/h3lp3rs/h3lp/MedicalCardAcivity.kt

```diff
@@ -0,0 +1,128 @@
+package com.github.h3lp3rs.h3lp
+
+import android.content.Intent
+import android.icu.util.Calendar
+import android.os.Build
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+import androidx.annotation.RequiresApi
+import androidx.core.widget.doOnTextChanged
+import com.google.android.material.textfield.TextInputLayout
+
+import android.widget.AutoCompleteTextView
+
+import android.widget.ArrayAdapter
+
+
+class MedicalCardAcivity : AppCompatActivity() {
+    @RequiresApi(Build.VERSION_CODES.N)
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_card)
+
+        createBirthField()
+        createHeightField()
+        createWeightField()
+        createBloodField()
+        createGenderField()
+    }
+
+
+    @RequiresApi(Build.VERSION_CODES.N)
+    private fun createBirthField() {
+        val birthTxt = findViewById<EditText>(R.id.medicalInfoBirthEditTxt)
+        val birthLayout = findViewById<TextInputLayout>(R.id.medicalInfoBirthTxtLayout)
+        birthTxt.doOnTextChanged { text, _, _, _ ->
+            when {
+                text!!.isEmpty() -> {
+                    birthLayout.error = null
+                }
+                text.toString().toInt() > Calendar.getInstance().get(Calendar.YEAR) -> {
+                    birthLayout.error = getString(R.string.yearTooRecent)
+                }
+                text.toString().toInt() < 1900 -> {
```

Comment:
Similarly to the tests, this should be a const val so that we understand its meaning